Vagrantfile

```diff
@@ -298,6 +310,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
                   ENV["CONTIV_CLUSTER_STORE"] || "etcd://localhost:2379",
                   ENV["CONTIV_DOCKER_VERSION"] || docker_version,
                   ENV['CONTIV_NODE_OS'] || "",
+                  ENV['CONTIV_V2PLUGIN_NAME'] || "contiv/v2netplugin:0.1",
```

Comment:
this is a default version for dev and the plugin is not pushed to hub. In the release script we should use the correct version and push it to the hub. Will add that in a following PR. Need to discuss that with you, joji and neelima